.github/workflows/tests.yaml

```diff
@@ -10,9 +10,12 @@ jobs:
   tests:
     runs-on: ubuntu-latest
     steps:
+      - name: Pull greenmail standalone
+        run:
+          docker pull greenmail/standalone:1.6.3
       - name: Start GreenMail testing server
-        run: |
-          docker run --rm -d -e GREENMAIL_OPTS='-Dgreenmail.setup.test.all -Dgreenmail.hostname=0.0.0.0 -Dgreenmail.auth.disabled -Dgreenmail.verbose' -p 3025:3025 -p 3110:3110 -p 3143:3143 -p 3465:3465 -p 3993:3993 -p 3995:3995 greenmail/standalone:1.6.2
```

Comment:
Why did you change this? The `--rm` is for docker to auto remove the container after the tests, and the `-d` is for the container to run as daemon (in background). The `-t` and `-i` are not needed because we are not in a TTY mode (the error is quite explicit btw: `the input device is not a TTY`).

Reply:
oh, I just copied the stuff from my execution file. I'll revert this

Reply:
It may solve the issue in fact!